qiskit_superstaq/superstaq_provider.py

```diff
@@ -36,7 +36,7 @@ class SuperstaQProvider(
 
         import qiskit_superstaq as qss
 
-        ss_provider = qss.superstaq_provider.SuperstaQProvider('MY_TOKEN')
+        ss_provider = qss.SuperstaQProvider('MY_TOKEN')
```

Comment:
```suggestion
        ss_provider = qss.SuperstaQProvider(api_key='MY_TOKEN')
```
(also do you remember why we didn't make api_key the first argument?)

Reply:
Done

Reply:
we should do the same thing for `css.Service`

Reply:
^ https://github.com/SupertechLabs/cirq-superstaq/issues/261